config/locales/en.yml

```diff
@@ -930,7 +930,7 @@ en:
         new_schedule: New Schedule
       name_and_timing_form:
         name: Name
-        orders_open: Orders open at
+        orders_open: Orders open
```

Comment:
"One item to look out for is an adjustment to an en.yml value - the English translation of "Orders Open At" was trimmed to "Orders Open" to match "Orders Close" and to keep this on one line with the asterisk."

This will be different in other languages. Translators will break this without knowing. But I assume having it broken in two lines is not too bad, or is it? Maybe a screenshot will help us decide.

Reply:
I agree - the English is actually the shortest translation; any other language will definitely break across the line, so it's not worth making the adjustment just for the English. I'll revert to the original value.